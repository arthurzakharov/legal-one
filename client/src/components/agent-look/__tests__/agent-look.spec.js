import React from 'react';
import renderer from 'react-test-renderer';
import AgentLook from '../agent-look';

describe('AgentLook: ', () => {
  test('render with all props passed', () => {
    const component = renderer
      .create(<AgentLook firstName="x" lastName="x" photo="x" identifier="x" onClick={() => 1} />)
      .toJSON();
    expect(component).toMatchSnapshot();
  });
});
