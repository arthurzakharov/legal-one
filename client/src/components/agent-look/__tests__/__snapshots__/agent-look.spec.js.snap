// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AgentLook:  render with all props passed 1`] = `
<div
  className="AgentLook"
  onClick={[Function]}
>
  <img
    alt="Logo-x"
    className="AgentLogo"
    src="x"
  />
  <h6>
    x x
  </h6>
</div>
`;
